src/dsv.js

```diff
@@ -5,9 +5,19 @@ var EOL = {},
     RETURN = 13;
 
 function objectConverter(columns) {
-  return new Function("d", "return {" + columns.map(function(name, i) {
-    return JSON.stringify(name) + ": d[" + i + "] || \"\"";
-  }).join(",") + "}");
+  try {
+    return new Function("d", "return {" + columns.map(function(name, i) {
+      return JSON.stringify(name) + ": d[" + i + "] || \"\"";
+    }).join(",") + "}");
+  } catch (ex) { // EvalError due to Content Security Policy.
+    return function(d) {
+      var result = {};
+      for (var i = 0; i < columns.length; i++) {
+        result[columns[i]] = d[i] || '';
```

Comment:
Minor nit re: formatting.

```suggestion
        result[columns[i]] = d[i] || "";
```

Also, is this `catch` branch covered by tests? I don't see any new tests here so I don't think the tests cover this? Test coverage I think should be there if this is to be merged.